content/en/docs/setup/install/helm/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Install with Helm
 linktitle: Install with Helm
-description: Install and configure Istio for in-depth evaluation.
+description: Instructions to install and configure Istio in a Kubernetes cluster using Helm.
```

Comment:
Maybe make a similar change in content/en/docs/setup/upgrade/helm/index.md.

Reply:
done